src/Views/FormPages/Confirmation.jsx

```diff
@@ -63,7 +63,7 @@ function Confirmation(props) {
 
                 <p><strong>{headings.confirmation.previous_name.label}</strong></p>
                 {!prevName && (
-                    <Alert type="info" headingLevel="h4" role="region" aria-live="polite" noIcon>
+                    <Alert type="info" headingLevel="h4" role="alert" aria-live="polite" noIcon>
```

Comment:
the other error needed a unique role for the alert message on the confirmation page